pyproject.toml

```diff
@@ -88,7 +87,9 @@ hf = [
   "datasets[vision]>=4.0.0",
   # https://github.com/pytorch/torchcodec/issues/640
   "datasets[audio]>=4.0.0 ; (sys_platform == 'linux' or sys_platform == 'darwin')",
-  "fsspec>=2024.12.0"
+  "fsspec>=2024.12.0",
+  # Until datasets solve the issue, run test_hf_audio test to see if this can be removed
+  "torch<2.9.0"
```

Comment:
C: I was getting:

```
E       RuntimeError: isString() INTERNAL ASSERT FAILED at "/Users/ec2-user/runner/_work/_temp/conda_environment_17551739977/lib/python3.12/site-packages/torch/include/ATen/core/ivalue_inl.h":2409, please report a bug to PyTorch. Expected String but got Blob
```

on running `test_hf_audio`. Most likely is is related at this point to a discrepancy in dependencies.